connector/src/main/java/org/geysermc/connector/utils/WebUtils.java

```diff
@@ -132,4 +136,41 @@ private static String connectionToString(HttpURLConnection con) throws IOExcepti
 
         return content.toString();
     }
+
+    /**
+     * Get the server log file and uploads it to mclo.gs
+     *
+     * @param log File to fetch
+     */
+    public static String mcLogs(Path log) throws IOException {
+        String mcversion = "unknown";
+        String userAgent = "unknown";
+        String version = "unknown";
+        //connect to api
+        URL url = new URL("https://api.mclo.gs/1/log");
+        URLConnection con = url.openConnection();
+        HttpURLConnection http = (HttpURLConnection) con;
+        http.setRequestMethod("POST");
+        http.setDoOutput(true);
+        //convert log to application/x-www-form-urlencoded
+        String content = "content=" + new BufferedReader(new InputStreamReader(Files.newInputStream(log.toRealPath()))).lines().collect(Collectors.joining("\n"));
+        byte[] out = content.getBytes(StandardCharsets.UTF_8);
+        int length = out.length;
+
+        //send log to api
+        http.setFixedLengthStreamingMode(length);
+        http.setRequestProperty("Content-Type", "application/x-www-form-urlencoded; charset=UTF-8");
+        http.setRequestProperty("User-Agent", userAgent + "/" + version + "/" + mcversion);
+        http.connect();
+        try (OutputStream os = http.getOutputStream()) {
+            os.write(out);
+        }
+        String br = new BufferedReader(new InputStreamReader(http.getInputStream()))
+                .lines()
+                .collect(Collectors.joining());
+        ObjectMapper om = new ObjectMapper();
```

Comment:
You should be able to re-use the `GeyserConnector.JSON_MAPPER` variable.

Reply:
Also I would rename `String br` to something else, since its the json as a String, the variable isnt a buffered reader